### DISCUSSION TITLE
how to check if agent has started span, if not how can I start span?

### DISCUSSION THREAD

**Discussion by annaik-mdsol** [open-telemetry/opentelemetry-java-instrumentation#12203] [Q&A] [answered]
I run my springboot app with otel agent, I want to ensure that agent creates span if not I want to create span for my app. I have created loggingContextFilter in springboot, how to see span starting and if not create one and then process doFilter?

**Comment by trask** [comment, marked as answer]
I think `Span.current().getSpanContext().isValid()` is what you're looking for
